app/database/MetricsDB.scala

```diff
@@ -40,10 +40,13 @@ class MetricsDB(val db: Database) {
       upsertPublishingMetric(Metric(metricOpt))
   }
 
-  def getForks: Either[ProductionMetricsError, List[ForkResponse]] =
-    awaitWithTransformation(db.run(forksTable.map(f => (f.issueDate.toDayDateTrunc, f.secondsUntilFork)).result)){ dbResult =>
+  def getForks(implicit filters: MetricsFilters): Either[ProductionMetricsError, List[ForkResponse]] =
+    awaitWithTransformation(db.run(
+      forksTable.map(f => (f.composerId, f.issueDate.toDayDateTrunc, f.secondsUntilFork))
+        .join(metricsTable).on(_._1 === _.composerId)
+        .filter(MetricsFilters.forksFilters).result)){ dbResult =>
       dbResult.flatMap {
-        case (dateOpt, timeOpt) => for {
+        case ((_, dateOpt, timeOpt), _) => for {
```

Comment:
I think I'd name the `_` even though you don't actually use them as I think it would make it more readable what's coming back from the db for future reference